msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/MessageForumsFilePickerServlet.java

```diff
@@ -167,6 +167,19 @@ protected void dispatch(HttpServletRequest req, HttpServletResponse res) throws
         RequestDispatcher dispatcher = getServletContext().getRequestDispatcher(target);
         dispatcher.forward(req, res);
 
+        //SAK-49399
```

Comment:
I wouldn't bother with these. We can track which jira ticket is associated with which commit, or line, using git blame.